qa/L0_s3_local/test.sh

```diff
@@ -118,6 +120,8 @@ if [ "$SERVER_PID" == "0" ]; then
     exit 1
 fi
 
+sleep 5 
```

Comment:
Why the server reports ready while it shouldn't?

Reply:
Filled a ticket to re-visit this issue.